src/UKHO.ADDS.EFS.Builder.S100/Pipelines/ExchangeSetPipelineContext.cs

```diff
@@ -39,5 +39,8 @@ public ExchangeSetPipelineContext(IConfiguration configuration, INodeStatusWrite
         public ExchangeSetJob Job { get; set; }
         public IEnumerable<BatchDetails> BatchDetails { get; set; }
         public string BatchId { get; set; }
+        public string ExchangeSetFileName { get; } = $"S100_ExchangeSet_{DateTime.UtcNow:yyyyMMdd}.zip";
+        public string ExchangeSetFilePath { get; set; } = "/usr/local/tomcat/ROOT/xchg/iicExchangeSetOutput";
```

Comment:
Avoid hardcoding ExchangeSetOutputDirectory inside ExchangeSetFilePath
ExchangeSetFilePath should be constructed at runtime by combining the root path and the output directory
```suggestion
        public string ExchangeSetFilePath { get; set; } = "/usr/local/tomcat/ROOT/xchg/";
```

Reply:
resolved